content/10.diagnostics.md

```diff
@@ -29,6 +29,16 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
+##### dPCR
+
+Digital PCR (dPCR) is a new generation of PCR technologies where a sample is partitioned into thousands of compartments, such as nanodroplets or nanowells, in each of which the PCR reaction takes place in.
+This leads to a digital read-out where each partition is either positive or negative for the nucleic acid being tested for.
+While dPCR equipment is not as common as RT-PCR, it generally has higher sensitivity and specificity than can be achieved with other PCR technologies [@doi: 10.3390/s18041271].
+High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
+Suo et al performed a double-blind evaluation of dPCR for SARS-CoV-2 detection on 57 samples--43 samples from suspected positive patients, and 14 from supposed convalescents--that all tested negative for SARS-CoV-2 using RT-PCR.
+In their hands, 33 out of 35 (94.3%) patients later clinically confirmed positive, as well as 9 out of 14 (64.2%) of supposed convalescents tested positive using dPCR [@doi:10.1101/2020.02.29.20029439].
```

Comment:
```suggestion
Despite the initial negative results, 33 out of 35 (94.3%) patients and 9 out of 14 (64.2%) supposed convalescents were later clinically confirmed positive.
All of these individuals tested positive using dPCR [@doi:10.1101/2020.02.29.20029439].
```
Is this accurate? I was trying to help make the three methods clearly distinct (originally negative, clinically positive, positive with this test as well).

Reply:
I think that's correct. The study was double-blind, so the clinical result was only revealed after the dPCR test.